src/olympia/lib/akismet/models.py

```diff
@@ -0,0 +1,111 @@
+import requests
+
+from django.conf import settings
+from django.db import models
+
+import olympia.core.logger
+from olympia.amo.models import ModelBase
+
+
+log = olympia.core.logger.getLogger('z.lib.akismet')
+
+
+class AkismetReport(ModelBase):
+    UNKNOWN = -1
+    HAM = 0
+    DEFINITE_SPAM = 1
+    MAYBE_SPAM = 2
```

Comment:
Are these values documented in the akismet api docs somewhere? If so, a link there would be nice

Reply:
No, they're constants I added. Lines 81-84 defines the logic.